shippable.yml

```diff
@@ -0,0 +1,34 @@
+# Language setting
+language: node_js
+
+# Version number
+node_js:
+  - 0.10.30
+
+# Create directories for test and coverage reports
+before_script:
+  - mkdir -p shippable/testresults
+  - mkdir -p shippable/codecoverage
+  - export XUNIT_FILE=shippable/testresults/results.xml
+  - npm install istanbul -g
+
+# Running the tests with npm
+script:
+  - grunt jshint
+  - grunt jscs
+  - istanbul cover _mocha -- --recursive -R xunit > shippable/testresults/results.xml
+  - istanbul report cobertura --dir  shippable/codecoverage/
+
+# Setup our environment with the IAM credentials
+# we're using IAM user "shippable"
+env:
+  global:
+    - SLACK_CHANNEL=#ci-cd
+    # SLACK_WEBHOOK
+    - secure: hXVEupfO2c9Wnx+lGw0tvN0A83XymNgi6x9QaHvAN8xUyeqAKzsCXjP9nGbWtkNPpAucCbo3qZz6zMWKncsZKiM2rnuYQ2mU+m95n5CNvqoD2JWPXyX8Sz2kUVoFcZ20EB05KVRTv9IHqv0dWkdlvafLCR2HVZ3UJJhH0gjQBGJqNlhbJKS8j+n3JV20AoIPg4sy3LGJWoZgfpGVXlhXANgt9RKw3QfZibeHGNAR97Q+StLZJG6VoK1+jv9U78CY4RzsmU93LYuZCI5V6hOGR0CLo7jv1ggpvGffdpRig1XtQWEC4cIUhRlqurxYaRReZnHoFIdjTmnjBQg2eUnoqQ==
```

Comment:
Good question. This is an encrypted key, that can only be un-encrypted by shippable when ran via our account.